cmd/ddev/cmd/config_test.go

```diff
@@ -375,6 +375,51 @@ func TestConfigSetValues(t *testing.T) {
 	assert.Equal("FOO=bar", app.WebEnvironment[1])
 	assert.Equal("SPACES=with spaces", app.WebEnvironment[3])
 	assert.Equal(webEnv, app.WebEnvironment[2])
+
+	// test docroot locations
+	testMatrix := []struct {
+		description string
+		input       string
+		expected    string
+		error       string
+	}{
```

Comment:
Thanks for this great set of tests. No need to change anything in this PR, but this test is already long and unreadable, it's OK to split into new and separate tests when working on something like this.